pyshelf/link_manager.py

```diff
@@ -68,3 +68,19 @@ def assign_single(self, artifact_path):
         ]
 
         self.context.link_list = link_list
+
+    def assign_metadata(self, metadata_path):
```

Comment:
That makes more sense, I fixed this but kept the rel the same. Let me know if they should be switched. I wasn't sure so I didn't do it. c83d14ca5df232f587e08b3e824d4142310ce8b7